docs/faq/troubleshooting/why-did-codacy-stop-commenting-on-pull-requests.md

```diff
@@ -1,8 +1,22 @@
 # Why did Codacy stop commenting on pull requests?
 
-Different reasons can cause Codacy to stop commenting on pull requests, but the most common is that the user who initially enabled the Git provider integration no longer has permissions on the repository or that the SSH key is no longer valid.
+Different reasons can cause Codacy to stop analyzing and commenting on pull requests, but the most common is that the user who initially enabled the Git provider integration no longer has permissions on the repository or that the SSH key is no longer valid.
 
-To fix this issue, we recommend that a user **with administrator permissions on the repository** re-enables the Git provider integration on Codacy:
+To fix this issue and also avoid future disruptions, Codacy recommends that you re-enable the Git provider integration on Codacy using a dedicated service account on your Git provider:
+
+1.  Create a service account on your Git provider exclusively dedicated to integrating Codacy with your repositories.
+
+    !!! note
+        The service account must have **administrator permissions** on the repositories to integrate with Codacy and **must not be shared by other systems** to ensure that Codacy doesn't hit the API rate limits of the Git provider when using this account.
```

Comment:
@jamescodacy following your feedack, I highlighted the fact that the service account must be for the exclusive use of the Codacy integration.